src/components/social-element/social-element.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+import PropTypes from "prop-types";
+
+import { classnames } from "../../helpers/utils";
+
+import * as styles from "./social-element.module.scss";
+
+const SocialElement = ({ className, links }) => {
+
+  return (
+    <div className={classnames(styles.container, className)}>
+      {links.map((item) => <a href={item?.path} target="_blank" rel="noopener noreferrer">{item?.icon}</a>,)}
```

Comment:
Why cand path and icon be optional?

Reply:
No, no, that was an error when I was creating the component.

Reply:
Done